tests/lifecycle/tests/lifecycle_scaling.go

```diff
@@ -0,0 +1,46 @@
+package tests
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalhelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalparameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/lifecycle/lifehelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/lifecycle/lifeparameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/namespaces"
+)
+
+// in order to run the test, export TNF_NON_INTRUSIVE_ONLY=false.
```

Comment:
Please set this variable automatically in your test and then remove it in After each section. Use  os.Setenv func.